src/pages/login.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { ReactElement } from 'react';
+import { NextPageContext } from 'next';
+import { parseCookies } from 'nookies';
+import Layout from '../layout/Layout';
+import ErrorSummary from '../components/ErrorSummary';
+import FormElementWrapper from '../components/FormElementWrapper';
+import { OPERATOR_COOKIE } from '../constants';
+import { ErrorInfo } from '../types';
+import { deleteCookieOnServerSide } from '../utils/index';
+
+const title = 'Login - Fares data build tool';
+const description = 'Login page of the Fares data build tool';
+
+interface LoginProps {
+    errors: ErrorInfo[];
+}
+
+const Login = ({ errors = [] }: LoginProps): ReactElement => {
+    return (
+        <Layout title={title} description={description}>
+            <main className="govuk-main-wrapper app-main-class" id="main-content" role="main">
+                <div className="govuk-grid-row">
+                    <div className="govuk-grid-column-two-thirds">
+                        <form action="/api/login" method="post">
+                            <ErrorSummary errors={errors} />
+                            <div className={`govuk-form-group ${errors.length > 0 ? 'govuk-form-group--error' : ''}`}>
+                                <div className="govuk-fieldset" aria-describedby="register-page-heading">
+                                    <legend className="govuk-fieldset__legend govuk-fieldset__legend--xl">
```

Comment:
This needs to be `govuk-fieldset__legend--l` instead of `govuk-fieldset__legend--xl`

Reply:
Now changed to  govuk-fieldset__legend--l